cli/src/generate/render.rs

```diff
@@ -255,6 +258,17 @@ impl Generator {
             .count();
     }
 
+    fn add_generator_version(&mut self) {
+        let version = if let Some(build_sha) = BUILD_SHA {
+            format!("{} ({})", BUILD_VERSION, build_sha)
+        } else {
+            BUILD_VERSION.to_string()
+        };
```

Comment:
This is copied out of `main.rs` because I wasn't sure of a good place to put it...

Reply:
It's fine imo